docker/ubuntu1604/Dockerfile

```diff
@@ -43,6 +43,9 @@ RUN groupadd -g 1002 jenkins && \
     useradd jenkins -s /bin/bash -g jenkins -m -u ${UID} -g 1002 -d /home/jenkins && \
     echo "jenkins ALL=NOPASSWD: ALL" >/etc/sudoers.d/jenkins
 
+#Creating TMP Directoy To Avoid permission Issues -- Bug in Xenial
+RUN mkdir /opt/tmp_docker; chmod -R 777 /opt/tmp_docker
```

Comment:
don't make this an extra step (layer) but create the directory below (around line 55) where it's used (and cleanup afterwards)